### PR TITLE
Add DBW to readme

### PR DESCRIPTION
**Pull request by ebidel**
R: @GoogleChrome/lighthouse

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -72,6 +75,22 @@ The audits and gatherers checked into the lighthouse repo are available to any c
...
 
+## Do Better Web
+
+**Do Better Web** is an initiative within Lighthouse to help web developers modernize their existing web applications. By running a set of tests, developers can discover new web platform APIs, become aware of performance pitfalls, and learn (newer) best practices. In other words, do better on the web!
+
+DBW is implemented as a set of standalone [gatherers](https://github.com/GoogleChrome/lighthouse/tree/master/lighthouse-core/gather/gatherers/dobetterweb), [audits](https://github.com/GoogleChrome/lighthouse/tree/master/lighthouse-core/audits/dobetterweb), and [custom run configuration](https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/config/dobetterweb.json) that can be run alongside the core Lighthouse tests.
```
Comment (brendankenny) [inline readme.md]: "and *a* custom run configuration"

File: readme.md [hunk 2]
```diff
@@ -72,6 +75,22 @@ The audits and gatherers checked into the lighthouse repo are available to any c
...
+
+To run DBW:
+
+    npm run dbw http://example.com
+
+which is a shorthand for:
+
+    npm run start -- --config-path=lighthouse-core/config/dobetterweb.json --disable-device-emulation --disable-cpu-throttling --disable-network-throttling --output=html --output-path=results.html`
+
+If you'd like to contribute, check the [list of issues](https://github.com/GoogleChrome/lighthouse/issues?q=is%3Aissue+is%3Aopen+label%3ADoBetterWeb) or proposal a new audit by filing an issue.
```
Comment (brendankenny) [inline readme.md]: "or propose a new audit"

File: readme.md (hunk 1 above)
Comment (ebidel) [inline readme.md]: done

File: readme.md (hunk 2 above)
Comment (ebidel) [inline readme.md]: done

**Pull request merged**